mailpoet/lib/API/JSON/API.php

```diff
@@ -100,13 +93,13 @@ public function setupAjax() {
       $this->setRequestData($_GET, Endpoint::TYPE_GET);
     }
 
-    $ignoreToken = (
```

Comment:
I wasn't able to figure out why we were ignoring tokens if captchas were enabled, and I couldn't see a downside to removing this check.